tests/python/test_sparse_linear_solver.py

```diff
@@ -24,6 +15,15 @@
                        0.0508907745638859, 0.290885785873098,
                        -0.518002544441220, 2.84177656043698
                    ]])
+"""
```

Comment:
did Pylint move this code block location?

Reply:
No, it is moved down because `yapf` and `isort` don't agree with each other, where `isort` tends to add a blank line between the import statements and the top comments but `yapf` doesn't think so.

Maybe I can make `yapf` ignore this line, let's see.